inductiva/simulators/README.md

```diff
@@ -0,0 +1,235 @@
+
+### Example
+
+```python
+import inductiva
+
+inductiva.api_key = "YOUR_API_KEY"
+
+# Set simulation input directory
+input_dir = "swash_example"
+
+# Initialize the Simulator
+simulator = inductiva.simulators.XBeach()
+
+# Run simulation with config files in the input directory
+task = simulator.run(input_dir=input_dir)
+```
+
+## GROMACS simulator
+
+GROMACS is a versatile simulator to perform molecular dynamics simulations. It is primarily designed for biochemical molecules like proteins, lipids and nucleic acids that have a lot of complicated bonded interactions, but since GROMACS is extremely fast at calculating the nonbonded interactions (that usually dominate simulations) many groups are also using it for research on non-biological systems, e.g. polymers and fluid dynamics.
+
+A single simulation of GROMACS via Inductiva API can comprise several steps - e.g., preparing the molecules, minimizing the energy of the system, running the simulation and post-processing. Hence, to configure a simulation of GROMACS the user may require several files. Moreover, GROMACS has specific commands to run certain tasks that already use the files in your input directory. 
+
+Below, we run the [protein solvation scenario]() with GROMACS and show how this is done in practice.
+
+### Example
+
+```python
+import inductiva
+
+inductiva.api_key = "YOUR_API_KEY"
+
+# Set simulation input directory
+input_dir = "molecules_water_box"
+
+# Set the simulation commands
+commands = [
+    {"cmd": "gmx solvate -cs tip4p -box {{ box_size }} -o conf.gro -p topol.top", "prompts": []},
+    {"cmd": "gmx grompp -f energy_minimization.mdp -o min.tpr -pp min.top -po min.mdp -c conf.gro -p topol.top", "prompts": []},
+    {"cmd": "gmx mdrun -s min.tpr -o min.trr -c min.gro -e min.edr -g min.log", "prompts": []},
+    {"cmd": "gmx grompp -f positions_decorrelation.mdp -o decorr.tpr -pp decorr.top -po decorr.mdp -c min.gro", "prompts": []},
+    {"cmd": "gmx mdrun -s decorr.tpr -o decorr.trr -x  -c decorr.gro -e decorr.edr -g decorr.log", "prompts": []},
+    {"cmd": "gmx grompp -f simulation.mdp -o eql.tpr -pp eql.top -po eql.mdp -c decorr.gro", "prompts": []},
+    {"cmd": "gmx mdrun -s eql.tpr -o eql.trr -x eql.xtc -c eql.gro -e eql.edr -g eql.log", "prompts": []}
+]
+
+# Initialize the Simulator
+simulator = inductiva.simulators.GROMACS()
+
+# Run simulation with config files in the input directory
+task = simulator.run(input_dir=input_dir, commands=commands)
+```
+
+## Large-scale simulations
+
+In all of the examples above, your simulations will run in the default set of machines we have available for everyone to use. These machines are not the most performant and mostly usefull for demo and testing purposes. To run large-scale simulations you can select your own machine group and only your simulations will run there.
```

Comment:
In all the examples above, your simulations will  run on our default set of machines, which are available for all users to use.  These machines are not the most performant and  are mostly useful for demo and testing purposes. To run large-scale simulations you have the option to choose your own dedicated machine group, where only your simulations will run.